src/sp_wrapper.c

```diff
@@ -0,0 +1,31 @@
+#include "php_snuffleupagus.h"
+#include "sp_config.h"
+
+ZEND_DECLARE_MODULE_GLOBALS(snuffleupagus)
+
+void sp_disable_wrapper() {
```

Comment:
Shouldn't we add guards to prevent multiple-hooks from happening, like in 00160b4430a7a02656fc07a7752d1c2d83e80871 ?

Reply:
It doesn't hook anything, so we should be fine.

Reply:
Right, worst case, it'll have a performance impact. We can live with this.